pkgs/development/python-modules/pydub/default.nix

```
@@ -21,23 +23,34 @@ buildPythonPackage rec {
     sha256 = "0xskllq66wqndjfmvp58k26cv3w480sqsil6ifwp4gghir7hqc8m";
   };
 
+  patches = [
+    # Fix codecs parsing with newer ffmpeg
+    # https://github.com/jiaaro/pydub/pull/713
+    (fetchpatch {
+      url = "https://github.com/jiaaro/pydub/commit/12fac9434b48a6ca5af6ae6db155816450facdc8.patch";
+      sha256 = "sha256-JJtnvlEL7pWxl5ICMiI3o4rW/1nbGD6mQMNCJ5XUYME=";
+    })
+    (substituteAll {
+      src = ./use-absolute-paths.diff;
+      ffmpeg = ffmpeg-full;
+    })
+  ];
+
   pythonImportsCheck = [
     "pydub"
     "pydub.audio_segment"
     "pydub.playback"
   ];
 
   nativeCheckInputs = [
-    ffmpeg-full
+    pytestCheckHook
   ];
 
-  checkPhase = ''
-    ${python.interpreter} test/test.py
-  '';
+  pytestFlagsArray = [ "test/test.py" ];
```

```suggestion
  pytestFlagsArray = [
    "test/test.py"
  ];
```

The only non-expanded list otherwise.